docs/logs/pipelines.md

```diff
@@ -0,0 +1,82 @@
+
+In case processor is not properly configured, nuber of errors will be displayed next to processor name and `Save Changes` button will be disabled.
+
+##### Filters
+Almost all processors have optional filters. With filters, you can select only a subset of events that the processor should be applied to. Events that do not match filters will be ignored.
+
+![Processor Filters](../../images/logs/pipelines/processor-filters.png)
+
+It uses our standard filter component with difference that you can also enter field that is not a keyword or even not yet defined. Another important difference is that you can use wildcards for values. It supports standard wildcards `*`, matches any number of characters, and `?`, matches a single character.
+This way you can e.g. drop events that have message field `*password*`.
+
+##### Sampling
+Events sampling is possible using Pipeline. It can be applied on all events or filters can be used to sample only some subset of events, e.g. only events from some service. Sample rate is expressed as percentage of events that should be stored. By default it is 100%, meaning no events will be dropped. Setting sampling to 10% means that only 1 in 10 events will be stored and 9 will be dropped.
+
+#### Preview
+Preview section is a helper tool to see how configured pipeline is working by running loaded or manually entered input through configured pipeline and showing differences.
+
+![Processor Filters](../../images/logs/pipelines/pipeline-preview.png)
+
+Differences are presented in a common way, line by line. Since our events are JSON objects, you can expand it and see which part of event was changed, removed or added.
+Input is automatically loaded from the App, respecting selected time, filters and number of documents. However, App might be empty, or documents in App might already be processed by pipeline so they do not have cases that you would like to test. In such cases, manual editing is the way to enter or adjust the input. You can toggle between editing and preview.
+
+![Edit Input](../../images/logs/pipelines/edit-input.png)
+
+At the moment, JSON is the only input format that is supported. Input editor will check each line if it is a valid JSON and report error if not. If there is a specif place where the error is, clicking on error icon will place cursor there.
+Once input is manually edited, a warning will be displayed if you try to reload data.
+
+![Reload Warning](../../images/logs/pipelines/reload-warning.png)
+
+#### Saving Pipeline
+Changes made to pipelines are not automatically saved. Once you made some changes and all processors are configured, `Save Changes` button will be enabled and pipeline configuration can be saved.
+
+![Save Pipeline](../../images/logs/pipelines/save-pipeline.png)
+
+The new configuration will not be applied immediatelly. It can take up to 5 min before you can see effects on your changes. Make sure you adjust field types using [Field Editor](./fields/#field-editor).
```

Comment:
```suggestion
The new configuration will not be applied immediately. It can take up to 5 minutes before your changes will take effect. Make sure you adjust field types using the [Field Editor](./fields/#field-editor).
```